frontend/src/components/auth/Login.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+    <v-content>
+                <v-flex
+                       lg6
+                       md6
+                       sm12
+                       xs12
+                >
+                    <v-container>
+                        <v-card-text class="login-container">
+                            <v-subheader
+                                    class="login-form-header"
+                            >Welcome to Metrica!</v-subheader>
+                            <v-form
+                                    ref="form"
+                            >
+                                <v-subheader
+                                        class="login-form-label"
+                                >Email</v-subheader>
+                                <v-text-field
+                                        name="email"
+                                        class="login-form-input"
+                                        v-model="email"
+                                        solo
+                                        type="text"
+                                        :rules="emailRules"
+                                        required
+                                >
+                                </v-text-field>
```

Comment:
consider using self closing tags where it's possible (see Vue Styleguide recomendation)